solutions/USACO feb16-milk-pails.mdx

```diff
@@ -7,11 +7,111 @@ author: Óscar Garries, Neo Wang, Nathan Gong
 
 [Official Analysis](http://www.usaco.org/current/data/sol_pails_silver_feb16.html)
 
+## Implementation (DFS)
+
+**Time Complexity:** $O(N^3)$
+
 <LanguageSection>
 	
 <CPPSection>
 
-## C++ Implementation (BFS)
+```cpp
+#include <bits/stdc++.h>
+
+using namespace std;
+
+bool vis[101][101][101];
```

Comment:
i think it's sufficient to just use two dimensions `bool vis[curX][curY]`

Reply:
This is the original floodfill/dfs solution. I was moving things around a lot so that's probably why this shows up as an addition.